Introduction/Getting started/External Linter/task.md

```diff
@@ -1,13 +1,21 @@
-## Enable External Linter
+IntelliJ Rust plugin doesn't detect all the errors. It relies on the Rust compiler to do that. While you are learning Rust, it is useful to see errors as you type. To have this behavior, we recommend to enable an external linter as follows.
```

Comment:
The IntelliJ Rust plugin doesn't detect all errors on its own; it relies on the Rust compiler for that. While learning Rust, it is helpful to see errors as you type. To achieve this, we recommend enabling an external linter as follows.